wallet/src/repository/wallet/mod.rs

```diff
@@ -240,6 +240,7 @@ where
             db_movements_to_update: Default::default(),
             transient_external_addresses: Default::default(),
             transient_internal_addresses: Default::default(),
+            stop_syncing: false,
```

Comment:
OMG, we are creating a monster... at some point we should refactor that struct. :rofl: